hip-service/Discovery/Patient/Helpers/FuzzyNameMatcher.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+
+namespace hip_service.Discovery.Patient.Helpers
+{
+    public static class FuzzyNameMatcher
+    {
+        public static int LevenshteinDistance(string s1, string s2)
+        {
+            var string1 = s1 == null ? "" : s1.Trim().ToLower();
+            var string2 = s2 == null ? "" : s2.Trim().ToLower();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4788558)